warning: in the working copy of 'pre210-working-v15/dsrc/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab', LF will be replaced by CRLF the next time Git touches it
diff --git a/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab b/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab
index 3102a60..252feec 100644
--- a/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/command/command_table.tab
@@ -1933,3 +1933,4 @@ unarmedHeadHit1	combat	normal	unarmedHeadHit1	failSpecialAttack			1.5	unarmedHea
 bodyShot2	combat	normal	bodyShot2	failSpecialAttack			1	bodyShot2	1	0	1	1	1	0	1	1	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	1	1	1	0	0	1	1	1	1	0	1	1	1	1	1	1	0	0	1	1	0	1	1	1	1	1	0	1	0	0	0	0	0	1	1	1	1	1	1		other	optional		2	0	391413347	0	0	0	0	1	PISTOL	NONE	defaultCooldownGroup	0	1.5	0		0	0	0
 bodyShot3	combat	normal	bodyShot3	failSpecialAttack			1	bodyShot3	1	0	1	1	1	0	1	1	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	1	1	1	0	0	1	1	1	1	0	1	1	1	1	1	1	0	0	1	1	0	1	1	1	1	1	0	1	0	0	0	0	0	1	1	1	1	1	1		other	optional		2	0	391413347	0	0	0	0	1	PISTOL	NONE	defaultCooldownGroup	0	1.5	0		0	0	0
 headShot3	combat	normal	headShot3	failSpecialAttack			2	headShot3	1	0	1	1	1	0	1	1	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	1	1	1	0	0	1	1	1	1	0	1	1	1	1	1	1	0	0	1	1	0	1	1	1	1	1	0	1	0	0	0	0	0	1	1	1	1	1	1		other	optional		2	0	391413347	0	0	0	0	1	RIFLE	NONE	defaultCooldownGroup	0	1.5	0		0	0	0
+melee1hBodyHit1	combat	normal	melee1hBodyHit1	failSpecialAttack			1.5	melee1hBodyHit1	1	0	1	1	1	0	1	1	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	1	1	1	0	0	1	1	1	1	0	1	1	1	1	1	1	0	0	1	1	0	1	1	1	1	1	0	1	0	0	0	0	0	1	1	1	1	1	1		other	optional		2	0	391413347	0	3	0	0	1	1HAND_MELEE	NONE	defaultCooldownGroup	0	1.5	0		0	0	0
warning: in the working copy of 'pre210-working-v15/dsrc/sku.0/sys.shared/compiled/game/datatables/combat/combat_data.tab', LF will be replaced by CRLF the next time Git touches it
diff --git a/sku.0/sys.shared/compiled/game/datatables/combat/combat_data.tab b/sku.0/sys.shared/compiled/game/datatables/combat/combat_data.tab
index 5b27c38..f03ba48 100644
--- a/sku.0/sys.shared/compiled/game/datatables/combat/combat_data.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/combat/combat_data.tab
@@ -1297,3 +1297,4 @@ unarmedHeadHit1	unarmedHeadHit1			-1	RIGHT_CLICK_SPECIAL	STANDARD	ATTACK	HEALTH
 bodyShot2	bodyShot2			-1	RIGHT_CLICK_SPECIAL	STANDARD	ATTACK	HEALTH	1				object/tangible/space/content_infrastructure/generic_egg_small.iff		-1.0	3.0	1	TARGETED_OBJECT	STANDARD	1	1	fire_1_special_single					fire_1_special_single									OFF	OFF	OFF	OFF	OFF	SINGLE_TARGET			0	64	0	0	3	0.0	1.0	0	0	0	0	0	100	0.0	40	0.0	0.0		0	0		0	0		0	0	1	0	0	0	0	0	0	0				PISTOL	RANGED_WEAPON	ENERGY	KINETIC				marksman	0		ACTION_NAME	0	0	1
 bodyShot3	bodyShot3			-1	RIGHT_CLICK_SPECIAL	STANDARD	ATTACK	HEALTH	1				object/tangible/space/content_infrastructure/generic_egg_small.iff		-1.0	3.0	1	TARGETED_OBJECT	STANDARD	1	1	fire_1_special_single					fire_1_special_single									OFF	OFF	OFF	OFF	OFF	SINGLE_TARGET			0	64	0	0	4	0.0	1.0	0	0	0	0	0	100	0.0	40	0.0	0.0		0	0		0	0		0	0	1	0	0	0	0	0	0	0				PISTOL	RANGED_WEAPON	ENERGY	KINETIC				marksman	0		ACTION_NAME	0	0	1
 headShot3	headShot3			-1	RIGHT_CLICK_SPECIAL	STANDARD	ATTACK	HEALTH	1				object/tangible/space/content_infrastructure/generic_egg_small.iff		-1.0	3.0	1	TARGETED_OBJECT	STANDARD	1	1	fire_1_special_single								fire_1_special_single						OFF	OFF	OFF	OFF	OFF	SINGLE_TARGET			0	64	0	0	3	0.0	1.0	0	0	0	0	0	100	0.0	40	0.0	0.0		0	0		0	0		0	0	1	0	0	0	0	0	0	0				RIFLE	RANGED_WEAPON	ENERGY	KINETIC				marksman	0		ACTION_NAME	0	0	1
+melee1hBodyHit1	melee1hBodyHit1			-1	RIGHT_CLICK_SPECIAL	STANDARD	ATTACK	HEALTH	1				object/tangible/space/content_infrastructure/generic_egg_small.iff		-1.0	3.0	1	TARGETED_OBJECT	STANDARD	1	1	counter_high_right				counter_high_right										OFF	OFF	OFF	OFF	OFF	SINGLE_TARGET			0	3	0	0	1.5	0.0	1.0	0	0	0	0	0	100	0.0	40	0.0	0.0		0	0		0	0		0	0	1	0	0	0	0	0	0	0				1HAND_MELEE	MELEE_WEAPON	KINETIC	KINETIC				brawler	0		ACTION_NAME	0	0	1
warning: in the working copy of 'pre210-working-v15/dsrc/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_overrides.tab', LF will be replaced by CRLF the next time Git touches it
diff --git a/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_overrides.tab b/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_overrides.tab
index ef51063..378a4f0 100644
--- a/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_overrides.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_overrides.tab
@@ -7,6 +7,7 @@ bodyShot2	0.5	1	0.5	HEALTH	1	50	RANGED
 bodyShot3	0.75	1.25	0.75	HEALTH	1	50	RANGED
 headShot2	0.5	0.5	1.5	MIND	1.5	5	RANGED
 headShot3	0.5	0.5	2.5	MIND	2	5	RANGED
+melee1hBodyHit1	0.5	0.5	0.625	HEALTH	1.5	25	INTENSITY
 legShot1	2.25	1.25	0.5	ACTION	2	25	NONE
 polearmLegHit1	1	0.5	0.5	ACTION	1.25	10	NONE
 polearmSpinAttack1	1.5	1	1	RANDOM	1.5	10	NONE
warning: in the working copy of 'pre210-working-v15/dsrc/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_spam.tab', LF will be replaced by CRLF the next time Git touches it
diff --git a/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_spam.tab b/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_spam.tab
index 081f413..e2b2f9c 100644
--- a/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_spam.tab
+++ b/sku.0/sys.shared/compiled/game/datatables/combat/precu_combat_spam.tab
@@ -6,6 +6,7 @@ bodyShot2	bodyshot
 bodyShot3	bodyshot
 headShot2	expertheadshot
 headShot3	masterheadshot
+melee1hBodyHit1	saimai
 legShot1	leg
 polearmLegHit1	legbruiser
 polearmSpinAttack1	limbsmasher
warning: in the working copy of 'pre210-working-v15/dsrc/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java', LF will be replaced by CRLF the next time Git touches it
diff --git a/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java b/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java
index 12e3bed..2a092c6 100644
--- a/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java
+++ b/sku.0/sys.server/compiled/game/script/systems/combat/combat_actions.java
@@ -14639,4 +14639,12 @@ public class combat_actions extends script.systems.combat.combat_base
         }
         return SCRIPT_CONTINUE;
     }
+    public int melee1hBodyHit1(obj_id self, obj_id target, String params, float defaultTime) throws InterruptedException
+    {
+        if (!combatStandardAction("melee1hBodyHit1", self, target, params, "", ""))
+        {
+            return SCRIPT_OVERRIDE;
+        }
+        return SCRIPT_CONTINUE;
+    }
 }
warning: in the working copy of 'pre210-working-v15/dsrc/sku.0/sys.server/compiled/game/script/test/precu_headshot1_fixture.java', LF will be replaced by CRLF the next time Git touches it
diff --git a/sku.0/sys.server/compiled/game/script/test/precu_headshot1_fixture.java b/sku.0/sys.server/compiled/game/script/test/precu_headshot1_fixture.java
index 64f91e7..d76351b 100644
--- a/sku.0/sys.server/compiled/game/script/test/precu_headshot1_fixture.java
+++ b/sku.0/sys.server/compiled/game/script/test/precu_headshot1_fixture.java
@@ -28,6 +28,7 @@ public class precu_headshot1_fixture extends script.base_script
     private static final String UNARMED_COMMAND = "unarmedHeadHit1";
     private static final String POLEARM_AREA_COMMAND = "polearmSpinAttack1";
     private static final String ONE_HAND_AREA_COMMAND = "melee1hSpinAttack1";
+    private static final String ONE_HAND_BODY_ONE_COMMAND = "melee1hBodyHit1";
     private static final String TWO_HAND_AREA_COMMAND = "melee2hSpinAttack1";
     private static final String POLEARM_CERTIFICATION = "cert_lance_staff_wood_s2";
     private static final String ONE_HAND_CERTIFICATION = "cert_sword_blade_rantok";
@@ -76,6 +77,8 @@ public class precu_headshot1_fixture extends script.base_script
         ROOT + ".originalPolearmAreaCommand";
     private static final String ORIGINAL_ONE_HAND_AREA_COMMAND =
         ROOT + ".originalOneHandAreaCommand";
+    private static final String ORIGINAL_ONE_HAND_BODY_ONE_COMMAND =
+        ROOT + ".originalOneHandBodyOneCommand";
     private static final String ORIGINAL_TWO_HAND_AREA_COMMAND =
         ROOT + ".originalTwoHandAreaCommand";
     private static final String ORIGINAL_POLEARM_CERTIFICATION =
@@ -251,6 +254,9 @@ public class precu_headshot1_fixture extends script.base_script
         boolean oneHandAreaCommandGranted =
             hasCommand(attacker, ONE_HAND_AREA_COMMAND) ||
             grantCommand(attacker, ONE_HAND_AREA_COMMAND);
+        boolean oneHandBodyOneCommandGranted =
+            hasCommand(attacker, ONE_HAND_BODY_ONE_COMMAND) ||
+            grantCommand(attacker, ONE_HAND_BODY_ONE_COMMAND);
         boolean twoHandAreaCommandGranted =
             hasCommand(attacker, TWO_HAND_AREA_COMMAND) ||
             grantCommand(attacker, TWO_HAND_AREA_COMMAND);
@@ -277,6 +283,8 @@ public class precu_headshot1_fixture extends script.base_script
             !hasCommand(attacker, POLEARM_AREA_COMMAND) ||
             !oneHandAreaCommandGranted ||
             !hasCommand(attacker, ONE_HAND_AREA_COMMAND) ||
+            !oneHandBodyOneCommandGranted ||
+            !hasCommand(attacker, ONE_HAND_BODY_ONE_COMMAND) ||
             !twoHandAreaCommandGranted ||
             !hasCommand(attacker, TWO_HAND_AREA_COMMAND) ||
             !polearmCertificationGranted ||
@@ -302,6 +310,8 @@ public class precu_headshot1_fixture extends script.base_script
                     hasCommand(attacker, POLEARM_AREA_COMMAND) +
                 " oneHandAreaCommand=" +
                     hasCommand(attacker, ONE_HAND_AREA_COMMAND) +
+                " oneHandBodyOneCommand=" +
+                    hasCommand(attacker, ONE_HAND_BODY_ONE_COMMAND) +
                 " twoHandAreaCommand=" +
                     hasCommand(attacker, TWO_HAND_AREA_COMMAND) +
                 " polearmCertification=" +
@@ -499,6 +509,12 @@ public class precu_headshot1_fixture extends script.base_script
         {
             revokeCommand(attacker, ONE_HAND_AREA_COMMAND);
         }
+        if (attackerRoot && hasObjVar(attacker, ORIGINAL_ONE_HAND_BODY_ONE_COMMAND) &&
+            getIntObjVar(attacker, ORIGINAL_ONE_HAND_BODY_ONE_COMMAND) == 0 &&
+            hasCommand(attacker, ONE_HAND_BODY_ONE_COMMAND))
+        {
+            revokeCommand(attacker, ONE_HAND_BODY_ONE_COMMAND);
+        }
         if (attackerRoot && hasObjVar(attacker, ORIGINAL_TWO_HAND_AREA_COMMAND) &&
             getIntObjVar(attacker, ORIGINAL_TWO_HAND_AREA_COMMAND) == 0 &&
             hasCommand(attacker, TWO_HAND_AREA_COMMAND))
@@ -721,6 +737,8 @@ public class precu_headshot1_fixture extends script.base_script
             hasCommand(player, POLEARM_AREA_COMMAND) ? 1 : 0);
         setObjVar(player, ORIGINAL_ONE_HAND_AREA_COMMAND,
             hasCommand(player, ONE_HAND_AREA_COMMAND) ? 1 : 0);
+        setObjVar(player, ORIGINAL_ONE_HAND_BODY_ONE_COMMAND,
+            hasCommand(player, ONE_HAND_BODY_ONE_COMMAND) ? 1 : 0);
         setObjVar(player, ORIGINAL_TWO_HAND_AREA_COMMAND,
             hasCommand(player, TWO_HAND_AREA_COMMAND) ? 1 : 0);
         setObjVar(player, ORIGINAL_POLEARM_CERTIFICATION,
@@ -816,6 +834,12 @@ public class precu_headshot1_fixture extends script.base_script
         {
             revokeCommand(attacker, ONE_HAND_AREA_COMMAND);
         }
+        if (hasObjVar(attacker, ORIGINAL_ONE_HAND_BODY_ONE_COMMAND) &&
+            getIntObjVar(attacker, ORIGINAL_ONE_HAND_BODY_ONE_COMMAND) == 0 &&
+            hasCommand(attacker, ONE_HAND_BODY_ONE_COMMAND))
+        {
+            revokeCommand(attacker, ONE_HAND_BODY_ONE_COMMAND);
+        }
         if (hasObjVar(attacker, ORIGINAL_TWO_HAND_AREA_COMMAND) &&
             getIntObjVar(attacker, ORIGINAL_TWO_HAND_AREA_COMMAND) == 0 &&
             hasCommand(attacker, TWO_HAND_AREA_COMMAND))
@@ -983,6 +1007,9 @@ public class precu_headshot1_fixture extends script.base_script
         boolean oneHandAreaCommandReady =
             hasCommand(attacker, ONE_HAND_AREA_COMMAND) ||
             grantCommand(attacker, ONE_HAND_AREA_COMMAND);
+        boolean oneHandBodyOneCommandReady =
+            hasCommand(attacker, ONE_HAND_BODY_ONE_COMMAND) ||
+            grantCommand(attacker, ONE_HAND_BODY_ONE_COMMAND);
         boolean twoHandAreaCommandReady =
             hasCommand(attacker, TWO_HAND_AREA_COMMAND) ||
             grantCommand(attacker, TWO_HAND_AREA_COMMAND);
@@ -1000,6 +1027,7 @@ public class precu_headshot1_fixture extends script.base_script
             cdefCertificationReady && pistolCdefCertificationReady &&
             polearmCommandReady && unarmedCommandReady &&
             polearmAreaCommandReady && oneHandAreaCommandReady &&
+            oneHandBodyOneCommandReady &&
             twoHandAreaCommandReady && polearmCertificationReady &&
             oneHandCertificationReady && twoHandCertificationReady &&
             attackerStanding && defenderStanding &&
@@ -1162,6 +1190,7 @@ public class precu_headshot1_fixture extends script.base_script
             hasObjVar(player, ORIGINAL_UNARMED_COMMAND) &&
             hasObjVar(player, ORIGINAL_POLEARM_AREA_COMMAND) &&
             hasObjVar(player, ORIGINAL_ONE_HAND_AREA_COMMAND) &&
+            hasObjVar(player, ORIGINAL_ONE_HAND_BODY_ONE_COMMAND) &&
             hasObjVar(player, ORIGINAL_TWO_HAND_AREA_COMMAND) &&
             hasObjVar(player, ORIGINAL_POLEARM_CERTIFICATION) &&
             hasObjVar(player, ORIGINAL_ONE_HAND_CERTIFICATION) &&
@@ -1238,6 +1267,8 @@ public class precu_headshot1_fixture extends script.base_script
             combat.canPerformAction(POLEARM_AREA_COMMAND, attacker) : -1;
         int canPerformOneHandArea = isIdValid(weapon) ?
             combat.canPerformAction(ONE_HAND_AREA_COMMAND, attacker) : -1;
+        int canPerformOneHandBodyOne = isIdValid(weapon) ?
+            combat.canPerformAction(ONE_HAND_BODY_ONE_COMMAND, attacker) : -1;
         int canPerformTwoHandArea = isIdValid(weapon) ?
             combat.canPerformAction(TWO_HAND_AREA_COMMAND, attacker) : -1;
         String fixtureWeapon = hasObjVar(attacker, FIXTURE_WEAPON) ?
@@ -1276,6 +1307,8 @@ public class precu_headshot1_fixture extends script.base_script
                 hasCommand(attacker, POLEARM_AREA_COMMAND) +
             " oneHandAreaCommand=" +
                 hasCommand(attacker, ONE_HAND_AREA_COMMAND) +
+            " oneHandBodyOneCommand=" +
+                hasCommand(attacker, ONE_HAND_BODY_ONE_COMMAND) +
             " twoHandAreaCommand=" +
                 hasCommand(attacker, TWO_HAND_AREA_COMMAND) +
             " polearmCertification=" +
@@ -1295,6 +1328,7 @@ public class precu_headshot1_fixture extends script.base_script
             " canPerformUnarmed=" + canPerformUnarmed +
             " canPerformPolearmArea=" + canPerformPolearmArea +
             " canPerformOneHandArea=" + canPerformOneHandArea +
+            " canPerformOneHandBodyOne=" + canPerformOneHandBodyOne +
             " canPerformTwoHandArea=" + canPerformTwoHandArea +
             " fixtureWeapon=" + fixtureWeapon +
             " fixturePistol=" + fixturePistol +

